src/app/app.module.ts

```diff
@@ -32,6 +32,7 @@ import { SystemModule } from './system/system.module';
 
 /** Main Routing Module */
 import { AppRoutingModule } from './app-routing.module';
+import { NotFoundComponent } from './not-found/not-found.component';
```

Comment:
Make sure that the import comes under the correct comment. Under the import for WebAppComponent, import this and add a proper comment like the one mentioned in the previous file.